ocs/agent/aggregator.py

```diff
@@ -167,6 +168,8 @@ def __init__(self, address, sessid, prov_id, frame_length=5*60, fresh_time=3*60)
         self.last_refresh = time.time() # Determines if
         self.last_block_received = None
 
+        self.log.warn("Recieved unxpected keyword argument(s): {kwarg}", kwarg=kwargs)
```

Comment:
We should include the agent address in the warning so people know what to change.

Reply:
Great suggestion! Added.